tera-templates/docsrs/about.html

```diff
@@ -0,0 +1,195 @@
+{% extends "base.html" -%}
+
+{%- block title -%} About Docs.rs {%- endblock title -%}
+
+{%- block body -%}
+    {%- set docsrs_repo = "https://github.com/rust-lang/docs.rs" -%}
+
+    <h1 id="crate-title">About Docs.rs</h1>
+    <div class="container about">
+        <p>
+            Docs.rs (formerly cratesfyi) is an
+            <a href="{{ docsrs_repo }}">open source</a>
+            documentation host for crates of the
+            <a href="https://www.rust-lang.org/">Rust Programming Language</a>.
+        </p>
+
+        <p>
+            Docs.rs automatically builds crates' documentation released on
+            <a href="https://crates.io/">crates.io</a>
+            using the nightly release of the Rust compiler.
+
+            {%- if rustc_version -%}
+            The current version of the Rust compiler in use is <code>{{ rustc_version }}</code>.
+            If you need a newer version of this compiler, check the
+            <a href="{{ docsrs_repo }}/issues">issues page</a>
+            and file a new issue if you don't see an existing request.
+            {%- endif -%}
+        </p>
+
+        <p>
+            The source code of Docs.rs is available on
+            <a href="{{ docsrs_repo }}">GitHub</a>. If
+            you ever encounter an issue, don't hesitate to report it! (And
+            thanks in advance!)
+        </p>
+
+        <p>
+            The README of a crate is taken from the <code>readme</code> field defined in
+            <code>Cargo.toml</code>. If a crate doesn't have this field,
+            no README will be displayed.
+        </p>
+
+        <h4>Global sandbox limits</h4>
+
+        <p>
+            All the builds on docs.rs are executed inside a sandbox with limited
+            resources. The current limits are the following:
+        </p>
+
+        {{ macros::crate_limits(limits=limits) }}
```

Comment:
`crate_limits` seems like it would only be used here, why is it a macro?

Reply:
It's also used on the build pages, it was much easier to pull into a component